modules/UI/util/UIUtil.js

```diff
@@ -111,6 +111,20 @@ import KeyboardShortcut from '../../keyboardshortcut/keyboardshortcut';
             title: this._getTooltipText.bind(this, element),
             html: true
         });
+
+        $(element).addClass('active-tooltip');
+    },
+
+    destroyTooltip: function (element) {
+        AJS.$(element).tooltip('destroy');
+    },
+
+    showTooltip: function (element) {
+        AJS.$(element).tooltip('show');
+    },
+
+    hideTooltip: function (element) {
+        AJS.$(element).tooltip('hide');
     },
 
```

Comment:
Could you please fix the PR, as we introduced new method in UIUtil: removeTooltip? And also add comments to the other added methods that are without jsdoc (showTooltip and hideTooltip).